package-lock.json

```
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "biodrop",
-      "version": "2.99.9",
+      "version": "2.99.17",
```

Profile changes doesn't require any changes `lock` files. So, please revert the changes from `package-lock.json` file or commit the below suggestion. 

```suggestion
      "version": "2.99.9",
```